integration/test_magic_folder.py

```diff
@@ -23,6 +23,70 @@ def test_alice_writes_bob_receives(magic_folder):
     return
 
 
+def test_alice_writes_bob_receives_multiple(magic_folder):
+    """
+    When Alice does a series of updates, Bob should just receive them
+    with no .backup or .conflict files being produced.
+    """
+    alice_dir, bob_dir = magic_folder
+
+    # first update
+    with open(join(alice_dir, "multiple"), "w") as f:
+        f.write("alice wrote this")
+
+    util.await_file_contents(join(bob_dir, "multiple"), "alice wrote this")
+    assert not exists(join(bob_dir, "multiple.backup"))
+    assert not exists(join(bob_dir, "multiple.conflict"))
+
+    # second update
+    time.sleep(2)
+    with open(join(alice_dir, "multiple"), "w") as f:
+        f.write("alice changed her mind")
+
+    util.await_file_contents(join(bob_dir, "multiple"), "alice changed her mind")
+    assert not exists(join(bob_dir, "multiple.backup"))
+    assert not exists(join(bob_dir, "multiple.conflict"))
+
+    # third update
+    time.sleep(2)
+    with open(join(alice_dir, "multiple"), "w") as f:
+        f.write("absolutely final version ship it")
+
+    util.await_file_contents(join(bob_dir, "multiple"), "absolutely final version ship it")
+    assert not exists(join(bob_dir, "multiple.backup"))
+    assert not exists(join(bob_dir, "multiple.conflict"))
+
+    # forth update, but both "at once" so one should conflict
+    time.sleep(2)
+    with open(join(alice_dir, "multiple"), "w") as f:
+        f.write("okay one more attempt")
+    with open(join(bob_dir, "multiple"), "w") as f:
+        f.write("...but just let me add")
+
+    bob_conflict = join(bob_dir, "multiple.conflict")
+    alice_conflict = join(alice_dir, "multiple.conflict")
+    bob_backup = join(bob_dir, "multiple.backup")
+    alice_backup = join(alice_dir, "multiple.backup")
+
+    found = None
+    start_time = time.time()
+    while time.time() - start_time < 30:
+        print("  waiting for conflict")
```

Comment:
They are always displayed when you run with `py.test -s` and never displayed otherwise. All the "flog" statements are captured by the "gather" fixture and dumped into a tmpfile